builder/docker/assemblyscript/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM suborbital/subo:dev as subo
 
 FROM node:16-buster-slim
-WORKDIR /root/runnable
+WORKDIR /usr/src/runnable
```

Comment:
Why was this path chosen? `/usr/*` paths are usually root owned as well and I don't see a `chown` anywhere for it. Does all of the writing happen in the `/tmp` dir set as `$HOME` in the Docker invocation?

Reply:
I...honestly don't remember where `/usr/src` came from, I definitely remember `/root` being an issue though.

The `/tmp` as HOME change came later for the Go/Swift images and I don't remember that being needed for other images, but I think this might vary depending on what's the UID/GID of the invoking user.